dom/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+       
+        .notificationClass {
+            background-color: green;
+            text-align: center;
+            display: flex;
+            justify-content: center;
+            border-radius: 20px;
+            align-items: center;
+            height: 100px;
+            color: white;
+            font-size: 40px;
+
+        }
+        .notificationClass2{
```

Comment:
2?  it's a bad practice
you have 2 almost identical classes
move common styles to a separate class and name it notification
rename current classes to success and error